projects/1.21/assets/arcane-abilities/arcane_abilities/lang/zh_cn.json

```
@@ -0,0 +1,152 @@
+  "tooltip.relics.fate_beacon.leveling_source.instant_return.description": "«信标标记»每传送50格增加1技能经验",
+
+  "item.arcane_abilities.shadow_lantern": "暗影灯笼",
+  "tooltip.relics.shadow_lantern.description": "遗物拥有者可以控制黑夜到来，并在世界的黑暗区域快速穿行。",
+  "tooltip.relics.shadow_lantern.ability.dusk_stealth": "薄暮潜行",
+  "tooltip.relics.shadow_lantern.ability.dusk_stealth.predicate.dark": "最小光照等级",
+  "tooltip.relics.shadow_lantern.ability.dusk_stealth.description": "光照等级低于 %1$s 时，使得佩戴者完全隐形，并在 %2$s 秒内使移动速度增加 %3$s%% 。",
+  "tooltip.relics.shadow_lantern.ability.light_beam": "炫目光束",
+  "tooltip.relics.shadow_lantern.ability.light_beam.description": "将你周围的黑暗变为炫目的亮光，以此照亮 %1$s 格范围内所有敌对生物。",
+  "tooltip.relics.shadow_lantern.ability.time_magic": "时间魔法",
+  "tooltip.relics.shadow_lantern.ability.time_magic.description": "每 %1$s 分钟一次使用机会，可立即将当前时间由白天切换成夜晚。",
+  "tooltip.relics.shadow_lantern.leveling_source.dusk_stealth.title": "薄暮潜行",
+  "tooltip.relics.shadow_lantern.leveling_source.dusk_stealth.description": "每次激活%2$s能力增加 %1$s 技能经验",
+  "tooltip.relics.shadow_lantern.leveling_source.time_magic.title": "时间魔法",
+  "tooltip.relics.shadow_lantern.leveling_source.time_magic.description": "每次激活%2$s能力增加 %1$s 技能经验",
+
+  "item.arcane_abilities.stellar_sphere": "星荧项链",
+  "tooltip.relics.stellar_sphere.description": "遗器将赋予你星辰的力量，夜晚时恢复拥有者的生命值并驱离周围的敌人，还可以召唤群星砸落于敌人头上造成伤害。",
+  "tooltip.relics.stellar_sphere.ability.stellar_radiance": "群星之辉",
+  "tooltip.relics.stellar_sphere.ability.stellar_radiance.description": "创造出星辰的领域，逐渐恢复遗器拥有者的生命值，并对 %1$s 格范围内敌人造成 %2$s 点伤害。",
+  "tooltip.relics.stellar_sphere.ability.stellar_ascension": "天灵飞升",
+  "tooltip.relics.stellar_sphere.ability.stellar_ascension.description": "以星能令你升入天空，给予暂时飞行的能力。飞行能力在«创造»模式下无效。",
+  "tooltip.relics.stellar_sphere.ability.stellar_shower": "星坠",
+  "tooltip.relics.stellar_sphere.ability.stellar_shower.description": "召唤群星如骤雨坠落，对 %1$s 格范围内敌人造成 %3$s 点伤害，持续 %2$s 秒。",
+  "tooltip.relics.stellar_sphere.leveling_source.stellar_ascension.title": "天灵飞升",
+  "tooltip.relics.stellar_sphere.leveling_source.stellar_ascension.description": "每次激活%2$s能力增加 %1$s 技能经验",
+  "tooltip.relics.stellar_sphere.leveling_source.stellar_shower.title": "星坠",
+  "tooltip.relics.stellar_sphere.leveling_source.stellar_shower.description": "每次激活%2$s能力增加 %1$s 技能经验",
+
+  "item.arcane_abilities.icy_crystal": "寒冰戒指",
+  "tooltip.relics.icy_crystal.description": "遗物使用者可以激发控制冰雪的潜能，从而发射寒冰弹射物或释放繁雪旋风，攻击并冻结敌人",
+  "tooltip.relics.icy_crystal.ability.snowy_whirlwind": "繁雪旋风",
+  "tooltip.relics.icy_crystal.ability.snowy_whirlwind.description": "在身旁创造出由雪花组成的涡旋，对 %2$s 格范围内敌人造成 %1$s 点寒冰伤害，并击退它们。",
+  "tooltip.relics.icy_crystal.ability.frosty_shackles": "冰霜镣铐",
+  "tooltip.relics.icy_crystal.ability.frosty_shackles.description": "在 %1$s 秒内，冻结 %2$s 格范围内的生物。",
+  "tooltip.relics.icy_crystal.ability.icy_projectiles": "寒冰弹射物",
+  "tooltip.relics.icy_crystal.ability.icy_projectiles.description": "发射大小为 %1$s 的寒冰弹射物，对前方的敌人造成 %2$s 点伤害，冷却时间： %3$s 秒。",

```

```suggestion
  "tooltip.relics.icy_crystal.ability.icy_projectiles.description": "发射大小为 %1$s 的寒冰弹射物，对敌人造成 %2$s 点伤害，冷却时间： %3$s 秒。",
```
与原文保持一致
**Fires an ice projectile of size %1$s, dealing %2$s damage to the enemy. Cooldown - %3$s seconds.**